src/main/java/uk/gov/di/ipv/service/ConfigurationService.java

```diff
@@ -1,28 +1,42 @@
 package uk.gov.di.ipv.service;
 
 import java.util.Optional;
+import software.amazon.lambda.powertools.parameters.ParamManager;
+import software.amazon.lambda.powertools.parameters.SSMProvider;
+import uk.gov.di.ipv.dto.CredentialIssuers;
+import uk.gov.di.ipv.helpers.CredentialIssuerLoader;
 
 public class ConfigurationService {
 
     private static final long DEFAULT_BEARER_TOKEN_TTL_IN_SECS = 3600L;
 
-    private static ConfigurationService configurationService;
+    private final SSMProvider ssmProvider;
 
-    public static ConfigurationService getInstance() {
-        if (configurationService == null) {
-            configurationService = new ConfigurationService();
-        }
-        return configurationService;
+    public ConfigurationService(SSMProvider ssmProvider) {
+        this.ssmProvider = ssmProvider;
+    }
+
+    public ConfigurationService() {
+        this.ssmProvider = ParamManager.getSsmProvider();
     }
 
     public boolean isRunningLocally() {
-        return Boolean.parseBoolean(System.getenv("IS_LOCAL")) ;
+        return Boolean.parseBoolean(System.getenv("IS_LOCAL"));
     }
 
     public String getAuthCodesTableName() {
         return System.getenv("AUTH_CODES_TABLE_NAME");
     }
 
+    public String getCredentialIssuerConfigParameterStoreKey() {
+        return System.getenv("CREDENTIAL_ISSUER_CONFIG_PARAMETER_STORE_KEY");
+    }
+
+    public CredentialIssuers getCredentialIssuers() {
+        String value = ssmProvider.get(getCredentialIssuerConfigParameterStoreKey());
```

Comment:
A small thing: could we inline this var on rename it to something other than value?

Reply:
Done